contracts/access/IAccessControlAdminRules.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts v4.9.0 (access/IAccessControlAdminRules.sol)
+
+pragma solidity ^0.8.0;
+
+import "./IAccessControl.sol";
+
+/**
+ * @dev External interface of AccessControladminRules declared to support ERC165 detection.
+ *
+ * _Available since v4.9._
+ */
+interface IAccessControlAdminRules is IAccessControl {
+    /**
+     * @dev Emitted when an `DEFAULT_ADMIN_ROLE` transfer is started, setting `newAdmin`
+     * as the next owner to be claimed after `delayedUntil` is met.
+     */
+    event AdminRoleChangeStarted(address indexed newAdmin, uint48 delayedUntil);
+
+    /**
+     * @dev Returns the address of the current `DEFAULT_ADMIN_ROLE` owner.
+     */
+    function owner() external view returns (address);
+
+    /**
+     * @dev Returns the timestamp in which the pending admin can claim the
+     * `DEFAULT_ADMIN_ROLE` role.
+     */
+    function delayedUntil() external view returns (uint48);
+
+    /**
+     * @dev Returns the address of the pending `DEFAULT_ADMIN_ROLE` owner.
+     */
+    function pendingAdmin() external view returns (address);
```

Comment:
I had the same feeling, but I wouldn't like to introduce the concept of `owner` to the whole contract.

We may create an ERC-173 interface in `/common` and implement it in both access control and ownable contracts.
This way we can keep the `admin()` naming here and allow of others to extend without adding significant overhead (or can be implemented by default with `function owner() ... { return admin(); }`)

Reply:
ERC-173 is already in `/interfaces`

Reply:
> Reference

I saw. Anyhow, we'll support only `owner()` with IERC5313 to avoid introducing a the concept of "owner" and make clear it's from another domain (IERC5313)

Reply:
I think @ernestognw's point is that this contract should have its own native terminology, and that it should not be `owner`, but that we can implement `owner()` in addition to the native interface for EIP-5313 compliance.

I'm also feeling unsure about "admin" here, particularly because `AccessControl` already has an "admin" concept and it means something different.

How would you feel about using the term `root` as in "root account"? A more straightforward option that could be okay is `defaultAdmin`.